README.md

```diff
@@ -17,14 +21,43 @@
     nvm use
     npm install -g yarn
     yarn
-    yarn develop
+    yarn dev
+    ```
+
+    Optionally provide secret environment variables (such as API auth) through the `now-secrets.json` file.
+
+    API credentials are only required to build the front-end if the wp.com instance is set to _private_.
+
+    ```sh
+    cp now-secrets.example.json now-secrets.json
+    ```
+
+    Expected format of now-secrets.json
+
+    ```json
+    {
+      "@neon-wp-client-secret": "wp-api-secret",
+      "@neon-wp-client-id": "wp-api-id",
+      "@neon-wp-user": "wp-user",
+      "@neon-wp-password": "wp-password"
+    }
+    ```
+
+    Available environment variables
+
+    ```sh
+    WORDPRESS_BASE_URL="wordpress.com base url"
+    WORDPRESS_CLIENT_SECRET="wordpress.com api client secret"
+    WORDPRESS_CLIENT_ID="wordpress.com api client id"
+    WORDPRESS_USER="wordpress.com username"
+    WORDPRESS_PASSWORD="wordpress.com password"
     ```
 
-1.  **Open the source code and start editing!**
+3.  **Open the source code and start editing!**
 
     Your site is now running at `http://localhost:8000`!
 
-    *Note: You'll also see a second link: `http://localhost:8000/___graphql`. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://www.gatsbyjs.org/tutorial/part-five/#introducing-graphiql).*
+    \_Note: You'll also see a second link: `http://localhost:8000/___graphql`. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://www.gatsbyjs.org/tutorial/part-five/#introducing-graphiql).\_
```

Comment:
Why the `\_` ?

Reply:
ah I think vscode made some automatic edits to the syntax - I'll swap it back